source/org/zfin/profile/AccountInfo.java

```diff
@@ -28,8 +28,8 @@ public class AccountInfo implements Serializable {
     @Column(name = "login", nullable = false)
     private String login;
 
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "zdb_id")
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
Let's test if there are any issues in the cases where we have a Person without an AccountInfo